### ISSUE TITLE
Ability to export plasma themes

### ISSUE THREAD

**Issue by nick87720z** [QtCurve/qtcurve#17]
This feature should be almost like current export to qt3/kde3 in Legacy settings tabs, but this one is not so legacy, so may be rename that tab to some thing like "Integration" or heh, "Alien".

Since plasma themes were almost never broken, and it is matter of writing SVG files, there should not be problems. When exported default theme name is "qtcurve" (if some custom unsaved configuration used) or "qtcurve-${configuration name}", in all cases user-editable.

It is really hard to find plasma theme, matching to favorite qt4 look. More likely they could first create plasma theme, than try to get as close as possible look sometimes using pixmap-based themes. With qtcurve and sometimes other engines (bespin) inversed order may be usual, since with their graphical configurator it is so easy to get wanted look without knowing of all file formats (svg graphics scheme in plasma is also some kind of format). For plasma it is especially hard if wanted theme should follow system colors, like colorbit does.

**Comment by yuyichao** [comment]
I totally agree that plasma themes are hard to configure (on the other hand, your plasma theme from your screenshots looks awesome) (To be honest, I even find QtCurve's configure dialog not so intuitive and was thinking of adding side-by-side previews or option-lookup by clicking to the configure GUI (although probably not anytime soon T_T)), and I think that a plasma-theme editor will be very helpful for creating awesome plasma themes. However, I'm not so sure if plasma themes can be derived from widget themes although some plasma themes may looks better (or worse) with certain widget themes.

IMHO, sth that makes sense to add are options to configure and export plasma themes (it would be better if we can hook into the plasma theme engine but I'm not so sure about that) which may or may not share some options with the widget style. However, I'm not so sure what options/functions should that include and any of your suggestions will be appreciated.

In general, I am only a programmer but not an artiest and therefore I am not an expert to decide how to make a good theme or make two themes fit better with each other. As a result, my long term goal is to make it as easy as possible for artiest to create good themes with QtCurve. I know that I'm still very far from that so it will be very helpful if people like you could provide some more specific suggestions on what options/functions are useful (patches and pull requests are even more welcome).

**Comment by nick87720z** [comment]
Did not mention it - theme on screenshot is colorbit :). It seems to be oxygen clone, improved by color awareness. But panel border seems barely blurred comparing to origin, another good theme with same capability is Aya.

Though, btw, for me panels are separately set to blue sora, which is naturelly blue.
